editor/src/components/canvas/dom-walker.ts

```diff
@@ -179,14 +179,24 @@ function getCachedAttributesComingFromStyleSheets(
   return value
 }
 
-function useResizeObserver(invalidatedSceneIDsRef: React.MutableRefObject<Set<string>>) {
+function useResizeObserver(
+  invalidatedPathsRef: React.MutableRefObject<Set<string>>,
+  invalidatedScenesRef: React.MutableRefObject<Set<string>>,
+  selectedViews: React.MutableRefObject<Array<ElementPath>>,
+  canvasInteractionHappening: React.MutableRefObject<boolean>,
+) {
   const resizeObserver = React.useMemo((): ResizeObserver | null => {
     if (ObserversAvailable) {
       return new ResizeObserver((entries: any) => {
-        for (let entry of entries) {
-          const sceneID = findParentScene(entry.target)
-          if (sceneID != null) {
-            invalidatedSceneIDsRef.current.add(sceneID)
+        if (canvasInteractionHappening.current) {
```

Comment:
I think this will no longer give images good size, because they might receive their correct size after loading

Reply:
oops, this is only while dragging